src/main/java/com/example/demo/mapper/AddressMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.mapper;
+
+import com.example.demo.dto.response.AddressResponseDto;
+import com.example.demo.persistence.entity.Address;
+import com.example.demo.dto.request.AddressRequestDto;
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
```

Comment:
можно сделать даже еще лучше
```suggestion
@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
```